README.md

```diff
@@ -55,14 +55,33 @@ The official **documentation and manpage** for `apx` are available at <https://d
 
 ## Other distros
 
-> Please consider to keep the project name as `apx` to avoid confusion for users.
+#Overview
+To use with another distro, you can compile the to the distro using ``go`` and copy the files to the needed paths.
```

Comment:
Compile the what ti the distro? Probably missing a "binary" or similar between "the" and "to".